src/App.js

```diff
@@ -0,0 +1,22 @@
+import {
+  BrowserRouter as Router, Routes, Route,
+} from 'react-router-dom';
+import BookList from './components/BookList';
+import Categories from './components/Categories';
+import Header from './components/Header';
+import './App.css';
+
+function App() {
+  return (
+    <Router>
+      <Header />
+      <Routes>
+        <Route path="/books" element={<BookList />} />
```

Comment:
- [ ] The Book component should be rendered on the root path "/", not "/books". Whenever the app loads the book component, without having to click the navbar menu.

- [ ] Kindly remove the BookForm component as it is empty and not being used.

Reply:
@DammyShittu 

![image](https://user-images.githubusercontent.com/89544285/236245344-06a1a704-63be-4e6f-a0ca-8e77df0cdcc5.png)